### PR TITLE
changed order and fixed index.html

### PR DESCRIPTION
**Pull request by tjaeschke**

### CODE REVIEW

**Comment by tjaeschke** [comment]
changed Openshift version to 3.9.0

File: doc/getting-started.md [hunk 1]
```diff
@@ -795,27 +777,42 @@ Open [Bitbucket](http://192.168.56.31:7990/), login with your crowd administrati
...
+Amend `ods-configuration/ods-core/sonarqube/ocp-config/sonarqube.env`
+and type
+
+``` bash
+tailor update
+
+``` 
+confirm with `y` and installation should start.
+
+After th installation has taken place, change to the Openshift Webconsole and start a sonarqube build.
```
Comment (michaelsauter) [inline doc/getting-started.md]: `s/th/the`
`s/Openshift/OpenShift`
`s/sonarqube/SonarQube`

**Review by michaelsauter — DISMISSED** [review]
Just minor nitpicks on naming consistency :D

File: doc/getting-started.md (hunk 1 above)
Comment (tjaeschke) [reply doc/getting-started.md]: @michaelsauter fixed. Would you please approve again?

**Review by michaelsauter — APPROVED** [review]

**Pull request merged**